_summaries/sutskever2014sequence_2.md

```diff
@@ -1,19 +1,79 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{Sutskever2014sequence}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: trowk
+score: 6
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+### What is the core idea?
+
+DNN are models that perform very well on a variety of different tasks. However,
+these models struggle with sequences, problems where the inputs and outputs are
+not fixed. Instead of DNNs, we can use two different LSTMs: one to encode the
```

Comment:
LSTM is a type of DNNs, the key contribution the authors propose is arranging LSTMs in this specific configuration for seq2seq tasks.

Reply:
Fixed this to specify that LSTMs are DNNs and that the LSTMs are used specifically to solve this task.